### ISSUE TITLE
Detect and display when new major versions of semver'ed import paths are available.

### ISSUE THREAD

**Issue by shurcooL** [shurcooL/Go-Package-Store#33]
For example, if I have `foo/bar.v4` in my GOPATH, and `foo/bar.v5` becomes available, perhaps Go Package Store is an excellent opportunity to let me know about that?

**Comment by Brasilikum** [comment]
Yea, you mean something like a big red label?

**Comment by shurcooL** [comment]
Yes, something like that, but it doesn't necessarily have to be red.
